CODEOWNERS

```
@@ -0,0 +1,4 @@
+/proposed/ @php-fig/secretaries
+/bylaws/ @php-fig/secretaries
+/proposed/security* @michaelcullum @php-fig/psr-9-10
```

Michael is included in that team, so it shouldn't be needed here

```suggestion
/proposed/security* @php-fig/psr-9-10
```